src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -55,13 +55,11 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
         1 - (sch * r / Sigma),
         -Sigma / (Delta * c2),
         -Sigma / c2,
-        -(
-            (r ** 2 + a ** 2 + (sch * r * (a ** 2) * (sin(theta) ** 2) / Sigma))
-            * (sin(theta) ** 2)
-        )
-        / c2,
+        (Delta * a ** 2 * sin(theta) ** 2 - (r ** 2 + a ** 2) ** 2)
+        * sin(theta) ** 2
+        / (Sigma * c2),
```

Comment:
I am pretty confident about the Kerr expression. It is directly copied from wikipedia(not that it's a very reliable source, but works fine) :D 

Please revert the changes here. Or leave it if your expression reduces to the same expression( BIG IF )